packages/server/src/integrations/tests/utils/mongodb.ts

```diff
@@ -0,0 +1,41 @@
+import { Datasource, SourceName } from "@budibase/types"
+import { GenericContainer, Wait, StartedTestContainer } from "testcontainers"
+
+let container: StartedTestContainer | undefined
+
+export async function start(): Promise<StartedTestContainer> {
```

Comment:
NAB: we should probably create a class/interface for this, as the hope is we expand this, would be nice to make sure this has a consistent interface/if we do build any generic tests all of these can be used under one interface easily.

Reply:
There is one but it's a little disconnected in how it's implemented: https://github.com/Budibase/budibase/blob/90a6fef9577130fb4536ca7fb4c1470d3ddc7cf7/packages/server/src/integrations/tests/utils/index.ts#L10